src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -108,9 +110,42 @@ const keriAcdcChangeHandler = async (
 const AppWrapper = (props: { children: ReactNode }) => {
   const dispatch = useAppDispatch();
   const authentication = useAppSelector(getAuthentication);
-  const [initialised, setInitialised] = useState(false);
   const [agentInitErr, setAgentInitErr] = useState(false);
 
+  const ACTIVITY_TIMEOUT = 60000;
+  let timer: NodeJS.Timeout;
+
+  useEffect(() => {
+    const handleActivity = () => {
+      clearTimeout(timer);
+      timer = setTimeout(() => {
+        dispatch(logout());
+      }, ACTIVITY_TIMEOUT);
+    };
+
+    // TODO: detect appStateChange in android and ios to reduce the ACTIVITY_TIMEOUT
+    // App.addListener("appStateChange", handleAppStateChange);
```

Comment:
Are we going to tackle this? I still think the lock time should be different in either case.